### DISCUSSION TITLE
Deskew Takes a long time

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#700] [Help]
image.Deskew(new Percentage(40));
I use the following code to correct the deviation, but it takes a long time, what is the problem？
Using the latest version Q8

**Comment by dlemstra** [comment]
The deskew method is not a fast method but `takes a long time` is relative. How long does it take? Maybe you could include a code sample and test image?

**Reply by jueqing1015** [reply]
@dlemstra 
[ENG001.zip](https://github.com/dlemstra/Magick.NET/files/4946543/ENG001.zip)
Elapsed time 3621 milliseconds
MagickImage image = new MagickImage("ENG001.tif");
            Stopwatch sw = new Stopwatch();
            sw.Start();
            image.Deskew(new Percentage(40));
            //image.Deskew(settings);
            sw.Stop();
            TimeSpan ts2 = sw.Elapsed;
            MessageBox.Show(ts2.TotalMilliseconds.ToString());

**Reply by dlemstra** [reply]
Deskew is also "slow" on my machine it takes around 2 seconds. The bigger the image the longer it takes. Also took a quick look to see if there was something to speed this up but I couldn't find something.
